microsoft/testsuites/cloud_hypervisor/ch_tests.py

```diff
@@ -70,8 +70,20 @@ def verify_cloud_hypervisor_integration_tests(
     ) -> None:
         hypervisor = self._get_hypervisor_param(node)
         ref = variables.get("cloudhypervisor_ref", "")
+        # below variable expects a comma separated list of full testnames
+        test_list_raw = variables.get("ch_integration_test_list", "")
```

Comment:
Make the name more consistent and simple, like `ch_tests_included`, `ch_tests_excluded`

Reply:
Changed.